pkg/odo/cli/experimental/debug/portforward.go

```diff
@@ -0,0 +1,170 @@
+package debug
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"time"
+
+	componentlabels "github.com/openshift/odo/pkg/component/labels"
+	"github.com/openshift/odo/pkg/config"
+	"github.com/openshift/odo/pkg/debug"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+
+	"github.com/openshift/odo/pkg/util"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	k8sgenclioptions "k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+
+	"k8s.io/kubernetes/pkg/kubectl/util/i18n"
+)
+
+// PortForwardOptions contains all the options for running the port-forward cli command.
+type PortForwardOptions struct {
+	Namespace  string
+	Address    []string
+	Ports      []string
+	contextDir string
+
+	PortForwarder debug.PortForwarder
+	StopChannel   chan struct{}
+	ReadyChannel  chan struct{}
+	*genericclioptions.Context
+	localConfigInfo *config.LocalConfigInfo
+}
+
+var (
+	portforwardLong = templates.LongDesc(i18n.T(`
```

Comment:
Why `i18n`? odo doesn't have multi-language support.

Reply:
removed

Reply:
but also shouldnt we?